src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -1258,19 +1253,19 @@ private void createMainTable() {
             public void actionPerformed(ActionEvent e) {
                 // need to close these here, b/c this action overshadows the responsible actions when the main table is selected
                 switch (mode) {
-                case SHOWING_NOTHING:
-                    frame.getGlobalSearchBar().endSearch();
-                    break;
-                case SHOWING_PREVIEW:
-                    getPreviewPanel().close();
-                    break;
-                case SHOWING_EDITOR:
-                case WILL_SHOW_EDITOR:
-                    entryEditorClosing(getCurrentEditor());
-                    break;
-                default:
-                    LOGGER.warn("unknown BasePanelMode: '" + mode + "', doing nothing");
-                    break;
+                    case SHOWING_NOTHING:
```

Comment:
@Siedlerchr I remember that you (Eclipse?) changed the indention of the switch statement directly in the opposite way. We should fix a style and stick to it. @koppor Any preference?